test_themes/static/src/js/navbar.js

```diff
@@ -1,30 +0,0 @@
-odoo.define('test_themes.website_selector', function (require) {
-'use strict';
-
-const websiteNavbarData = require('website.navbar');
-const { registry } = require("@web/core/registry");
-
-const TestThemesWebsiteSelector = websiteNavbarData.WebsiteNavbarActionWidget.extend({
-    /**
-     * @override
-     */
-    start: function () {
-        this.$('.js_multi_website_switch[data-toggle]').tooltip({
-            html: true,
-            placement: 'left',
-            delay: {show: 100, hide: 0},
-        });
-        return this._super(...arguments);
-    },
-
-});
-
-registry.category("website_navbar_widgets").add("TestThemesWebsiteSelector", {
-    Widget: TestThemesWebsiteSelector,
-    selector: '#website_switcher',
-});
-
-return {
-    TestThemesWebsiteSelector: TestThemesWebsiteSelector,
-};
-});
```

Comment:
I don't mind this being reimplemented post merge but maybe we could preserve some of the history by at least not removing file that will be reintroduced later ? :/

At least this JS file, I would:
- Have a commit which, in this case, just comments the previous code, with a sentence at the top stating "TODO this will be reimplemented following the frontend UI being moved to the backend". That way we never have a point in the history with the feature being gone without any explication.

Post merge:
- Have a commit which only *moves* the file to the right location to be added in the backend assets (-> preserve history, technically, I would have liked that for all website files that were dropped in the community PR too but too late I guess :wink:)
- Have a commit which uncomments and adapts the code.